src/api/routes/company.js

```diff
@@ -101,4 +102,45 @@ module.exports = (app) => {
                 throw err;
             }
         });
+
+    /**
+     * Enables a previously disabled company
+     */
+    router.put("/:companyId/enable",
+        or([
+            authMiddleware.isCompany,
+            authMiddleware.isAdmin,
+            authMiddleware.isGod
+        ], { status_code: HTTPStatus.UNAUTHORIZED, error_code: ErrorTypes.FORBIDDEN, msg: ValidationReasons.INSUFFICIENT_PERMISSIONS }),
+        validators.enable,
+        (req, res, next) => companyMiddleware.validCompany(req.params.companyId)(req, res, next),
+        async (req, res, next) => {
+            try {
+                await new OfferService().enableByCompany(req.params.companyId);
+                const company = await (new CompanyService()).enable(req.params.companyId);
+                return res.json(company);
```

Comment:
I think we should send an email notifying the user about this action, similar to what is done in the block/unblock scenario